handlers/main.yml

```diff
@@ -1,10 +1,17 @@
 # SPDX-License-Identifier: MIT
 ---
-# Put handlers for tasks here.
 
-# Example:
-- name: template handler to restart services
-  service:
-    name: "{{ item }}"
-    state: restarted
-  loop: "{{ __template_services }}"
+- name: reboot the managed host to apply crypto policies changes
+  reboot:
```

Comment:
The `reboot` module takes an optional `test_command` that tells the role when the machine is at the stage where it can continue.  Is there some such command for crypto policies that you can use to determine if the system has reached the stage where the crypto policies have been applied after reboot?  Perhaps it isn't necessary in this case and you can use the default behavior of the `reboot` module.

Reply:
There is nothing special crypto policies would need to do after the reboot -- all of the changes are made before reboot and reboot just makes sure all the services and applications dynamically loading crypto libraries are started with a new configuration.